Example/PaymentSheet Example/PaymentSheet Example/ExampleCustomDeferredCheckoutViewController.swift

```diff
@@ -0,0 +1,332 @@
+//
+//  ExampleCustomDeferredCheckoutViewController.swift
+//  PaymentSheet Example
+//
+//  Created by Nick Porter on 3/27/23.
+//
+
+import Foundation
+//  Note: Do not import Stripe using `@_spi(STP)` in production.
+//  This exposes internal functionality which may cause unexpected behavior if used directly.
+// TODO(porter) Remove STP usage before GA
+@_spi (STP) import StripePaymentSheet
+import UIKit
+
+class ExampleCustomDeferredCheckoutViewController: UIViewController {
+    @IBOutlet weak var buyButton: UIButton!
+    @IBOutlet weak var paymentMethodButton: UIButton!
+    @IBOutlet weak var paymentMethodImage: UIImageView!
+
+    @IBOutlet weak var hotDogQuantityLabel: UILabel!
+    @IBOutlet weak var saladQuantityLabel: UILabel!
+    @IBOutlet weak var hotDogStepper: UIStepper!
+    @IBOutlet weak var saladStepper: UIStepper!
+    @IBOutlet weak var subtotalLabel: UILabel!
+    @IBOutlet weak var salesTaxLabel: UILabel!
+    @IBOutlet weak var totalLabel: UILabel!
+    @IBOutlet weak var subscribeSwitch: UISwitch!
+
+    private var paymentSheetFlowController: PaymentSheet.FlowController!
+    private let baseUrl = "https://stripe-mobile-payment-sheet-custom-deferred.glitch.me"
```

Comment:
Let's add a comment like 
// View the backend code here: https://glitch.com/edit/#!/stripe-mobile-payment-sheet-custom-deferred